ext_emconf.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+$EM_CONF[$_EXTKEY] = [
+    'title' => 'Search Core: Facebook',
+    'description' => 'Search core implementation for Facebook.',
+    'category' => 'be',
+    'clearCacheOnLoad' => 1,
+    'constraints' => [
+        'depends' => [
+            'typo3' => '7.6.0-8.7.99',
+            'php' => '7.0.0-7.2.99',
+            'search_core' => '',
+        ],
+        'conflicts' => [],
```

Comment:
Remove if we don't have conflicts?

Reply:
I'll keep it, I'm even not sure whether it's required.